File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_5/Python/main.py
print("Introduce el primer valor: ")
iNum1 = int(input())
print("Introduce el segundo valor: ")
iNum2 = int(input())
print("Introduce el tercer valor: ")
iNum3 = int(input())
print("El número mayor es = ", end='', flush=True)
if iNum1 >= iNum2 and iNum1 >= iNum3:
    print(iNum1)
else:
    if iNum2 >= iNum1 and iNum2 >= iNum3:
        print(iNum2)
    else:
        if iNum3 >= iNum1 and iNum3 >= iNum2:
            print(iNum3)

File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_5/C++/main.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    int iNum1, iNum2, iNum3;
    
    cout << "Introduce el primer valor: " << endl;
    cin >> iNum1;
    cout << "Introduce el segundo valor: " << endl;
    cin >> iNum2;
    cout << "Introduce el tercer valor: " << endl;
    cin >> iNum3;
    cout << "El número mayor es = ";
    if (iNum1 >= iNum2 && iNum1 >= iNum3) {
        cout << iNum1 << endl;
    } else {
        if (iNum2 >= iNum1 && iNum2 >= iNum3) {
            cout << iNum2 << endl;
        } else {
            if (iNum3 >= iNum1 && iNum3 >= iNum2) {
                cout << iNum3 << endl;
            }
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}
